### DISCUSSION TITLE
Return File Instead of Output

### DISCUSSION THREAD

**Discussion by cpsdakota** [adnanh/webhook#510] [Q&A] [answered]
I use this tool to execute python scripts and it runs great. I use the option to return the result to my web browser.

I would like instead to return a file that the client downloads, is there an option to do this?

**Comment by adnanh** [comment, marked as answer]
Something like this?

```
[
        {
                "id": "test",
                "execute-command": "/home/adnan/test.sh",
                "response-headers": [
                        {
                        "name": "Content-Type",
                        "value": "aplication/octet-stream"
                        },
                        {
                        "name": "Content-Transfer-Encoding",
                        "value": "binary"
                        },
                        {
                        "name": "Content-Disposition",
                        "value": "attachment; filename=\"testhook.txt\""
                        }
                ],
                "include-command-output-in-response": true
        }
]
```

**Reply by cpsdakota** [reply]
Brilliant! Thank you so much!

**Reply by cpsdakota** [reply]
I have two follow up questions. Is it possible to get "testhook.txt" from "generated_configs\testhook.txt". In other words, get the file one directory deep.

Is there a way to extract the user's input entered in the "pass-arguments-to-command" in order to dynamically create the filename I want to return?   "generated_configs\<variable>.txt"
